src/simulator/subscriber.rs

```diff
@@ -39,12 +39,12 @@ impl Subscriber {
             }
         }
 
+        let topic = config.topic_format.replacen("{pub_id}", "+", 1);
+        let topic = topic.replacen("{data_type}", "+", 1);
```

Comment:
shouldn't this be same as the data type provided in the argument instead of '+'?

Reply:
i wanted to avoid the string type validation here so kept it `+` as during a single run there would be publish on only 1 data type, but with `ValueEnum` is now trivial.